src/command_modules/azure-cli-acr/setup.py

```diff
@@ -14,7 +14,7 @@
     logger.warn("Wheel is not available, disabling bdist_wheel hook")
     cmdclass = {}
 
-VERSION = "2.2.7"
+VERSION = "2.2.8"
```

Comment:
Check if this needs to be `2.2.9` based on whether `2.2.8` is already released (I see dev already has `2.2.8`). If not, you need to merge the `history.rst` as per this [commit ](https://github.com/Azure/azure-cli/commit/a93391639134617536190e7eca57710fa6ce524b)

Reply:
Updated to 2.2.9 :)